lib/ohai/plugins/freebsd/filesystem.rb

```diff
@@ -39,6 +40,23 @@
       end
     end
 
+    # Grab filesystem inode data from df
+    so = shell_out("df -iP")
+    so.stdout.lines do |line|
+      case line
+      when /^Filesystem\s+Size/
```

Comment:
I don't see why we have this pattern everywhere, but you want `when /^Filesystem/` here.